motoko/encrypted-notes-dapp-vetkd/README.md

```diff
@@ -24,54 +18,47 @@ This example uses an **insecure** implementation of [the proposed vetKD system A
 
 ### Prerequisites
 
+This example requires an installation of:
+
+- [x] Install the [IC SDK](https://internetcomputer.org/docs/current/developer-docs/setup/install/index.mdx).
 - [x] Clone the example dapp project: `git clone https://github.com/dfinity/examples`
 
-### Step 1: Choose which implementation to use by setting a respective environment variable.
+## Step 1: Choose which implementation to use by setting a respective environment variable.
    
 For **Motoko** deployment use:
 
 ```sh
 export BUILD_ENV=motoko
 ```
 
-### Step 2: To generate `$BUILD_ENV`-specific files run:
+## Step 2: To generate `$BUILD_ENV`-specific files run:
 
 ```sh
 sh ./pre_deploy.sh
 ```
 
-### Step 3: [Install dfx](https://sdk.dfinity.org/docs/quickstart/local-quickstart.html). 
-
-Please keep in mind the `dfx` CLI currently only runs on Linux and macOS.
-
-### Step 4: Install npm packages from the project root:
+## Step 3: Install npm packages from the project root:
 
 ```sh
 npm install
 ```
 _Note_: see [Troubleshooting](#troubleshooting) in case of problems.
 
-### Step 5: In case `dfx` was already started before, run the following:
+## Step 4: Run in a separate window:
 
 ```sh
-dfx stop
-rm -rf .dfx
+dfx start --clean
 ```
 
-### Step 6: Run in a separate window:
+> ![TIP]
+> If you see an error `Failed to set socket of tcp builder to 0.0.0.0:8000`, make sure that the port `8000` is not occupied, e.g., by the previously run Docker command (you might want to stop the Docker daemon whatsoever for this step).
 
-```sh
-dfx start --clean
-```
-:::info
-If you see an error `Failed to set socket of tcp builder to 0.0.0.0:8000`, make sure that the port `8000` is not occupied, e.g., by the previously run Docker command (you might want to stop the Docker daemon whatsoever for this step).
-:::
 
-### Step 7: Install a local [Internet Identity (II)](https://wiki.internetcomputer.org/wiki/What_is_Internet_Identity) canister:
+## Step 5: Install a local [Internet Identity (II)](https://wiki.internetcomputer.org/wiki/What_is_Internet_Identity) canister:
+
+> ![TIP]
```

Comment:
This doesn't render correctly: It [seems](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts) this should be `> [!TIP]`

Reply:
Thanks, fixed